lifecycle-operator/controllers/common/schedulinggate/handler.go

```diff
@@ -1,4 +1,4 @@
-package common
+package schedulinggate
```

Comment:
The name should have the plural (like the previous type and [K8s](https://github.com/kubernetes/kubernetes/blob/246d363ea4bab2ac99a938d0cee73d72fc44de45/pkg/scheduler/framework/plugins/schedulinggates/scheduling_gates.go#L17))
```suggestion
package schedulinggates
```